detection_rules/schemas/definitions.py

```diff
@@ -85,7 +86,7 @@
 TimelineTemplateId = NewType('TimelineTemplateId', str, validate=validate.OneOf(list(TIMELINE_TEMPLATES)))
 TimelineTemplateTitle = NewType('TimelineTemplateTitle', str, validate=validate.OneOf(TIMELINE_TEMPLATES.values()))
 UUIDString = NewType('UUIDString', str, validate=validate.Regexp(UUID_PATTERN))
-
+BuildingBlockType = NewType('building_block_type', str, validate=validate.Regexp(BUILDING_BLOCK_TYPE))
```

Comment:
For simple validation, we have just used literal (such as `Maturity`)

```suggestion
BuildingBlockType = Literal['default']
```